src/views/home/home.tsx

```diff
@@ -50,7 +50,14 @@ export function Home(props: { modalDisclosure?: UseDisclosureProps }) {
   return (
     <Layout.Main>
       <SlideFade in>
-        <Container maxW={{ base: "auto", md: "container.sm" }}>
+        <Container
+          maxW={{
+            base: "auto",
+            md: "container.sm",
+            lg: "container.md",
+            xl: "container.lg",
+          }}
+        >
```

Comment:
The Home page was artificially limited to "container.sm" size. Now it scales up with the display.